tests/yast2_gui/yast2_control_center.pm

```diff
@@ -298,12 +322,19 @@ sub run {
     start_sudo;
     start_user_and_group_management;
     start_hypervisor;
-    if (check_var('DISTRI', 'sle')) {
+
+    if (is_sle) {
         start_add_system_extensions_or_modules;
         start_kernel_dump;
         start_common_server_certificate;
         start_ca_management;
-        start_wake_on_lan;
+        if (sle_version_at_least '15') {
+            # No wol on sle 15 ATM
+            record_soft_failure 'bsc#1059569';
+        }
+        else {
+            start_wake_on_lan;
+        }
```

Comment:
IMHO it looks cleaner if that is handled within the `start_wake_on_lan` method. Also this should allow to check for the error condition first as well

Reply:
Right, here we can do proper soft-fail without workaround: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/579 Please, merge new needle.